Adding link to new accessibility guide and distributing cloud service… (#329)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/style_guidelines/accessibility.adoc

[[accessibility]]
= Accessibility

> This section highlights some guidelines for writing accessible content. For full information about writing accessible content at Red Hat, see link:https://redhat-documentation.github.io/accessibility-guide/[_Getting started with accessibility for writers_].

Reviewer: ```suggestion
For full information about writing accessible content at Red Hat, see link:https://redhat-documentation.github.io/accessibility-guide/[_Getting started with accessibility for writers_]. Here is an overview of the accessibility guidelines.
```
Trying to omit "This section..." self-referential language, as mentioned elsewhere in this doc :-)
